packages/storybook/stories/VideoGallery/snippets/PPTLiveSharingFromViewer.snippet.tsx

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Is there a difference between screenshare snippet vs pptlive snippet just on component? If not, I think we should hide this code snippet

Reply:
No difference.